### PR TITLE
fix: reject if initialize is called more than once

### PR DESCRIPTION
**Pull request by tyanbiao**

### CODE REVIEW

File: android/src/main/java/io/wifi/p2p/WiFiP2PManagerModule.java [hunk 1]
```diff
@@ -90,6 +90,7 @@ public void onGroupInfoAvailable(WifiP2pGroup group) {
   @ReactMethod
   public void init(Promise promise) {
     if (manager != null) { // prevent reinitialization
+      promise.reject("You should only call initialize once.");
```
Comment (kirillzyusko) [inline android/src/main/java/io/wifi/p2p/WiFiP2PManagerModule.java:93]: Such signature is not deprecated? I thought it requires code + error message 🤔

Comment (kirillzyusko) [reply android/src/main/java/io/wifi/p2p/WiFiP2PManagerModule.java:93]: image

I think you can use `0x2` code 🙂

**Comment by kirillzyusko** [comment]
@tyanbiao wanted to ask you - are you going to submit more PRs? If not, then I'll prepare new `3.6.0` release :)

**Comment by tyanbiao** [comment]
> @tyanbiao wanted to ask you - are you going to submit more PRs? If not, then I'll prepare new `3.6.0` release :)

I've fixed it. Thank you! 🤗

**Pull request merged**